app/models/tweet.js

```diff
@@ -0,0 +1,13 @@
+import DS from 'ember-data';
+
+let Tweet =  DS.Model.extend({
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).